wpeverest/user-registration
# UR-1252 Tweak - Compatibility with newly introduced ThemeGrill Agency Plan.

### All Submissions:

* [x] Have you followed the [User RegistrationContributing guideline](https://github.com/wpeverest/everest-forms/blob/master/.github/CONTRIBUTING.md)?





### Changes proposed in this Pull Request:
A new ThemeGrill Agency Plan has been created which has yearly and lifetime subscriptions. Since a new plan has been added, the license of this plan must work with User Registration. This PR provides compatibility with the newly introduced ThemeGrill Agency Plan.

### How to test the changes in this Pull Request:

1. Activate User Registration Pro with the license of the ThemeGrill Agency plan.
2. Try to install addons from the Addons Page, Form Templates while creating a new Form, from settings area and From the addon requiring fields like Captcha, Repeater, Authorize net, Likert, etc. ( PS. Refresh Addons and Templates while doing that )
3. Check if you encounter any errors while doing this.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Compatibility with the newly introduced ThemeGrill Agency Plan.
